helm/opendj/README.md

```diff
@@ -111,3 +106,56 @@ with an alternate implementation.
 If you are benchmarking on a cloud provider make sure you use an SSD storage class as the directory is very sensitive 
 to disk performance.
 
+## Initializing a DS cluster from a previous backup (experimental)
+
+Backups to gcs are placed in the a directory structure that looks like this:
+
+`gs://backup-bucket/backup-root/{namespace}/{instance}/yy/mm/dd/`
+
+For example:
+
+`gs://forgeops/dj-backup/10m/sandbox/userstore/2018/05/31`
+
+Where 'sandbox' is the namespace name, and 'userstore' is the instance.
+
+The restore script will recurse through the `yy/mm/dd` to find the most recent backup. Within each day folder, there is a full backup and a a number of incrementals. The restore will restore the most recent incremental and full backup for that day.
```

Comment:
s/through the `yy/mm/dd`/through the `yy/mm/dd` directory/
s/day folder/day's directory/
s/incrementals/incremental backups/
s/The restore/The restore operation/